### ISSUE TITLE
Crash using pnpx (Don't know if it matters)

### ISSUE THREAD

**Issue by kebabbbb** [prisma/prisma#19986]
Hi Prisma Team! The following command just crashed.

## Command

`migrate dev`

## Versions
      
| Name        | Version            |
|-------------|--------------------|
| Platform    | debian-openssl-3.0.x| 
| Node        | v20.3.1            | 
| Prisma CLI  | 4.16.1             | 
| Engine      | b20ead4d3ab9e78ac112966e242ded703f4a052c| 

## Error
```
RustPanic: RuntimeError: panicked at 'where objects cannot be empty', query-engine/schema/src/build/input_types/objects/filter_objects.rs:195:9
```

**Comment by Jolg42** [comment]
Could you share your Prisma schema?

You mention `migrate dev` though I suspect this is actually happening when generating the Prisma Client.

Could you try running the following and post the output here?
- `npx prisma -v`
- `npx prisma generate`

**Comment by jkomyno** [comment]
Hey @kebabbbb, would you take a look at the question @Jolg42 asked in the previous comment, and please let us know about the missing information? This will help us identifying the problem. Thanks!

**Comment by jkomyno** [comment]
Hey @kebabbbb, could you please provide more information, as asked above? Thanks!

**Comment by kebabbbb** [comment]
Oh hi, sorry I just saw this. I really sorry, but the computer that I was using like crashed and my stuff got wiped, so I cant really answer it. Again, my apologies.

**Comment by Jolg42** [comment]
Then let's close this until we can get more information, it might be fixed in 5.0.0 based on this similar error report and comment https://github.com/prisma/prisma/issues/20345#issuecomment-1649806456
